frontend/src/components/shared/VirtualDropdown/VirtualDropdown.tsx

```diff
@@ -56,18 +52,18 @@ const VirtualDropdown = React.forwardRef((props: any, ref) => {
 
   return (
     <StyledList ref={ref}>
-      <FixedSizeList
+      <VariableSizeList
         width="100%"
         height={height}
         itemCount={children.length}
         itemData={children}
         itemKey={(index: number, data: { props: OptionListProps }[]) =>
           data[index].props.item.value
         }
-        itemSize={LIST_ITEM_HEIGHT}
+        itemSize={() => LIST_ITEM_HEIGHT}
       >
         {FixedSizeListItem}
-      </FixedSizeList>
+      </VariableSizeList>
```

Comment:
I'm not sure I understand the shift to VariableSizeList if we're still returning the same number for all items? I assume there's some reason, perhaps a comment?

Reply:
That was wrong. That was a test. Changed it back 😊